### ISSUE TITLE
A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_Keyboard:The focus order is not logical after the "select subscription" dropdown has been closed. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9232]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags:#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#MacOs;#MAS2.4.3;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.15.0-insider
Commit: 3a8b74a
Date: 2020-02-11T05:49:41.181Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.3.0
Mac OS:10.15.3

**Additional Details:**
Mac OS:10.15.3
MAS References:MAS2.4.3

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Connections from left menu.
4. Select Azure, navigate to + button & hit enter.
5. Navigate to Add account button in right side window & hit enter.
5. Tab to the "select subscription" button.

**Actual:**
On navigating to the button "select subscription" press enter an option list opens on escape it is closed and next tab key focus moves to controls under the commonly used heading 

**Expected:**
The focus after closing the options list should be on the button "select subscription".

**User Impact:**
Keyboard users will not be able to navigate to the items under the account after a subscription has been selected.

**Attachment link for Reference:**
[A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_9232.zip](https://github.com/microsoft/azuredatastudio/files/4322641/A11y_SQLAzureDataStudio_AddAzureAccount_AddAzureAccount_9232.zip)

**Comment by alanrenmsft** [comment]
not sure whether we can control where the focus goes to

**Comment by alanrenmsft** [comment]
this is a vscode popup we are leveraging, please find a way for vscode to repro the issue and open  a bug there, then resolve this one as upstream.

**Comment by aaomidi** [comment]
https://github.com/microsoft/vscode/pull/92969

**Comment by alanrenmsft** [comment]
@aaomidi not sure how long will it take for vscode to fix the issue, I am thinking of making the change directly in ADS now. there might be a merge conflict, but that is easy to resolve. what do you think?

**Comment by aaomidi** [comment]
We can do it and put the SQL CARBON EDIT on top of it.

**Comment by alanrenmsft** [comment]
@aaomidi yes, please do.

**Comment by BhanuPrakashGit** [comment]
Verified the bug, issue is not repro'ing.
GithubTags:#Closed;
PFA for reference
[9232_closed.zip](https://github.com/microsoft/azuredatastudio/files/10742224/9232_closed.zip)

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-MAC-Feb2020;
